src/modules/likeCount.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+import { addLikeCount, getLikeCount } from './involvementApi';
+
+export const pokedex = 110;
+
+export function addlikes() {
+  const likebtn = document.querySelectorAll('.heart');
+  likebtn.forEach((btn) => {
+    btn.addEventListener('click', async () => {
+      await addLikeCount(parseInt(btn.getAttribute('id')));
+
+      // update like display
+      const response = await getLikeCount();
+      const item = response.find(
+        (element) => element.item_id === parseInt(btn.getAttribute('id')),
+      );
+      const likeDisplay = document.querySelector(`#span-${btn.getAttribute('id')}`);
+      likeDisplay.innerHTML = item.likes;
+      console.log(response);
```

Comment:
- [ ] **[Optional]** I suggest you remove this console log to avoid printing a sensitive data to the console.